main.php

```diff
@@ -57,6 +57,10 @@ function vipgoci_help_print( $argv ) {
 		PHP_EOL .
 		'PHP Linting configuration:' . PHP_EOL .
 		"\t" . '--lint=BOOL                    Whether to do PHP linting. Default is true.' . PHP_EOL .
+		"\t" . '--lint-only-modified-files=BOOL   Whether to limit lint scan to run against only modified or new' . PHP_EOL .
+		"\t" . '                               files in the PR to be scanned. Default is true. It can be ' . PHP_EOL .
+		"\t" . '                               modified via options file ("' . VIPGOCI_OPTIONS_FILE_NAME . '") placed in' . PHP_EOL .
```

Comment:
I think we should let it suffice to document in the README file that this option can be overriden via options file, see [here](https://github.com/automattic/vip-go-ci/#configuration-via-repository-config-file).

Reply:
So, just to check if I understood correctly, are you suggesting removing the explanation from the helper?

Reply:
I mean just this text here:

> It can be modified via options file ("' . VIPGOCI_OPTIONS_FILE_NAME . '") placed in ...

I think it is better documented in the README file.

Reply:
Since this code is not actually adding the functionality logic, I'm planning to add it to the readme in the logic PR. What do you think?

Reply:
Sounds good to me :+1:

Reply:
https://github.com/Automattic/vip-go-ci/pull/215